### DISCUSSION TITLE
my github account has been hacked

### DISCUSSION THREAD

**Discussion by adriannsyahn** [community/community#126251] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

my github account was hacked please help me, the previous email was adriiansyahn@gmail.com now it has been changed by hackers to queen55kozeyjrg@hotmail.com please help me to recover my github
![github account hacked](https://github.com/community/community/assets/171056596/0e14071d-536f-4d05-8e63-bb6a4cd87689)

**Comment by UjjwalSaini07** [comment]
u enable 2 fa Authentication or download any backup code from github

**Comment by UjjwalSaini07** [comment]
please marks as answered my text it help me alot

**Comment by ExpiredWater** [comment]
Hey @adriannsyahn ,

You should look to contact GitHub support about your issue. Please see [here](https://docs.github.com/en/support/contacting-github-support) for more details. You should [create a support ticket.](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket#about-support-tickets). Quoted from this page: You can contact GitHub Support to report account, security, and abuse issues.

I wish you the best of luck on recovering your account.

**Comment by ebndev** [comment, marked as answer]
Hi @adriannsyahn,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
